package/Dockerfile.shipyard-dapper-base

```diff
@@ -44,7 +44,7 @@ ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH} PATH=/go/bin:/root/.l
 # - Any unnecessary packages and executables
 RUN dnf -y install --nodocs --setopt=install_weak_deps=False \
                    gcc git-core curl moby-engine make golang kubernetes-client \
-                   findutils moreutils upx jq gitlint procps-ng \
+                   findutils moreutils upx jq gitlint procps-ng pip\
```

Comment:
What do we need pip for?

Reply:
I imagine this is required so that `kind.sh` can set up the new Jinja2 tools; please add the reason to the commit message.

Reply:
Done.